training-net/src/Repositories/Database/DataBaseContext.cs

```diff
@@ -0,0 +1,19 @@
+#region Using
+using Microsoft.EntityFrameworkCore;
+using MvcMovie.Models;
+#endregion
+
+namespace MvcMovie.Repositories.Database
+{
+    public class DataBaseContext : DbContext
+    {
+        public DataBaseContext(DbContextOptions<DataBaseContext> options) : base(options) {}
+
+        public DbSet<Models.Database.Movie> Movies { get; set; }
```

Comment:
with "using MvcMovie.Models" you dont need to put all the namespace name here.
`public DbSet<Movie> Movies` should work